installer/README.md

```diff
@@ -93,16 +93,31 @@ Single Tenant mode will create tenant which will be default tenant to use while
 
 ## Tests
 * Prerequisites:
+  * bats
+    ```
+    git clone https://github.com/bats-core/bats-core.git && cd bats-core && ./install.sh /usr/local
```

Comment:
In previous PR you add this line to CI script, to task: `Run imm test`, but i think it should be part of installing prerequisites